client/boot/index.js

```diff
@@ -404,7 +407,7 @@ function reduxStoreReady( reduxStore ) {
 			document.getElementsByClassName( 'wp-singletree-layout' ).length
 		);
 
-		const singleTreeSections = [ 'theme', 'themes' ];
+		const singleTreeSections = [ 'browsehappy', 'theme', 'themes' ];
```

Comment:
I wish. 😕  The trouble here is that `/signup` is also set to be `isomorphic` in `sections.js` (even tho you could argue it isn't) because of #5823 (in a gist, that allows us to have no custom/section-specific routes in `server/pages`, but have them where they belong -- at section level).

I mean we can try and see what happens if we add `signup` here 😄